build.md

```diff
@@ -57,6 +57,19 @@ Install the build dependencies.
   * Build the binary.
   * Build the docker image with the binary.
 
+* To build multi-arch images, you need to have `buildx` installed. You can follow these [instructions](https://github.com/docker/buildx#installing).`
+  * NDM multi-arch images are built using QEMU mode. If you're on linux, run :
+    * docker run --rm --privileged linuxkit/binfmt:v0.8
+  * Set the value of envirionment variable of `IMAGE_ORG` based on your user name at Dockerhub:  
```

Comment:
```suggestion
  * Set the value of envirionment variable of `IMAGE_ORG` based on your user name at Dockerhub:  This is required because `buildx` cannot store the multiarch image locally, instead pushes to the registry after building.
```